tenseal/cpp/tensors/ckkstensor.cpp

```diff
@@ -273,7 +274,7 @@ shared_ptr<CKKSTensor> CKKSTensor::op_plain_inplace(
         Plaintext plaintext;
         for (size_t i = start; i < end; i++) {
             this->tenseal_context()->encode<CKKSEncoder>(
-                operand_data.at(i), plaintext, this->_init_scale);
+                operand_data[i], plaintext, this->_init_scale);
```

Comment:
Can't we use the same API as for this->_data? namely `flat_at` on the operand instead of fetching all the data vector first? Just to make sure we are iterating over the same positions from both vectors, we are sure `flat_at` will return the same position for both of them, even if the implementation changes in the future.